tests/utils.go

```diff
@@ -2084,12 +2084,14 @@ func GetGuestAgentUserData() string {
 	ipv6UserDataString := ""
 	if netutils.IsIPv6String(dnsServerIP) {
 		ipv6UserDataString = fmt.Sprintf(`sudo ip -6 addr add fd10:0:2::2/120 dev eth0
-                             sudo ip -6 route add default via fd10:0:2::1 src fd10:0:2::2
+                             for i in {1..20}; do sudo ip -6 route add default via fd10:0:2::1 src fd10:0:2::2 && break || sleep 0.1; done
```

Comment:
20 x 0.1 = 2 second, 
2 seconds are enough ?
we used 5 so far, maybe worth to be on the safe side, giving more time ?

Reply:
Yes, it is more than enough per my tests.
If it is not enough, we should see the tests failing.